scripts/changelog.js

```diff
@@ -84,7 +84,14 @@ function prereleaseTemplate(items) {
 
 [See the full changelog](${items.fullChangelogUrl})
 
-This is a release candidate for v${stableVersion(items.version)} to ensure maximum stability for public release.
+${prereleaseType(items.version) === "pre" ?
```

Comment:
I would suggest we switch this round.

So, if it's "rc" then it's a release candidate. Otherwise it's a pre-release.

My reasoning is basically we will never have something else refer to a release candidate (as that is 1 very specific thing) whereas a pre-release, we may well add extra terminology for certain other types of pre-release (beta, alpha, etc) and if we do that, then it'll by default show up in here as "pre-release" (which is true) rather than currently it would show as "release candidate" (which is untrue).

Now, I know we probably won't end up doing that because I know what your opinions are on those terms, etc. But I just thought it's better to make the more specific text (rc) only when it's definitely an rc, and the more generic (pre) for all other pre-releases.

Reply:
Suuuure, I was not putting as much thought into it as you, but yes, that makes sense. I like the idea of order of the code being order of releases (pre _then_ rc) but that's about it. Will switch.